docs/content/manual/_incoming/2274-vm-import.md

```diff
@@ -0,0 +1,246 @@
+---
+title: VM Import/Migration
+---
+
+* Related issues: [#2274](https://github.com/harvester/harvester/issues/2274) [Feature] VM Import/Migration
+
+## Category: 
+* Virtual Machine
+
+## Test Information
+Test Environment: 
+* 1 node harvester on local kvm machine
+* Harvester version: v1.1.0-rc1
+* Vsphere: 7.0
+* Openstack: Simulated using running devstack
+* Download kubeconfig for harvester cluster
+
+## Environment Setup
+
+1. Prepare Harvester master node
+2. Prepare vsphere setup (or use existing setup)
+3. Prepare a devstack cluster (Openstack 16.2)
```

Comment:
@ibrokethecloud is this the `stable/yoga` branch?
As I was trying to provision a `stable/yoga` instance via some #cloud-config [here](https://gist.github.com/irishgordo/b12f35e02b8e4c52620f546b189f5757)

Reply:
Doesn't need to be stable yoga per say, as even most recent version of openstack works well "zed"